claims_option.go

```diff
@@ -0,0 +1,31 @@
+package jwt
+
+import "time"
+
+// ClaimsValidationOptions represents options that can be used for claims validation
+type ClaimsValidationOptions struct {
```

Comment:
I would call this `ValidatorOptions`, similar to the `ParserOptions` we have

Reply:
Renamed